.github/CODEOWNERS

```diff
@@ -12,3 +12,28 @@
 # This is being deprecated.
 /Tools/RemoteConsole/ @o3de/sig-testing-reviewers
 
+# SIG-Core areas of ownership
+/Code/LauncherUnified/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/Tools/AssetProcessor/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/CrashHandler/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/Framework/AzCore/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/Framework/AzFramework/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/Framework/AzGameFramework/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Code/Framework/AzToolsFramework/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Gems/CrashReporting/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Gems/LmbrCentral/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Gems/Profiler/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Gems/Prefab/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
+/Gems/ImGui/ @o3de/sig-core-reviewers @o3de/sig-core-maintainers
```

Comment:
Until sig-simulation is created, all the physics gems are owned by sig-core, please add them to the list.

````
/Gems/PhysX/
/Gems/PhysXDebug/
/Gems/NvCloth/
/Gems/Blast/
````

I think it'd be great if automated tests owned by sig-core were also part of this list. Here are the physics ones:

````
/AutomatedTesting/Gem/PythonTests/Physics
/AutomatedTesting/Gem/PythonTests/NvCloth
/AutomatedTesting/Gem/PythonTests/Blast
````

From the sig-core list presented it seems the following automated tests folder should be owned by sig-core too:

````
/AutomatedTesting/Gem/PythonTests/Prefab
/AutomatedTesting/Gem/PythonTests/assetpipeline   // Some subfolders inside assetpipeline might be owned by sig-content, maybe worth clarifying with them.
````

Reply:
I can make that change.